### DISCUSSION TITLE
is it possible to join uniswap Labs as internship student ?

### DISCUSSION THREAD

**Discussion by DoTheBestToGetTheBest** [Uniswap/interface#7452] [General]
Hello everyone hope all are goods. Please can you tell me if it's possible to join as internship student for everything related developement ?

Via remote

I don't want to be paid or something. Just get an internship.

**Comment by RustRanger** [comment]
hi. I'm interested in the same question

**Reply by AlekseyFomm** [reply]
Likewise...

**Comment by Durann21** [comment]
Good idea on how to find experience.

**Comment by EddyHoganu** [comment]
interesting information
